pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -457,3 +462,16 @@ const (
 	// DBTypePostgres is postgres
 	DBTypePostgres DBTypes = "postgres"
 )
+
+// AutoscalingTypes is a string enum type for specify the types of autoscalling supported.
+type AutoscalingTypes string
+
+// These are the valid DB types:
+const (
+	// AutoscalingTypeDefault is hpa
+	AutoscalingTypeDefault AutoscalingTypes = "default"
```

Comment:
I think it should be `AutoscalingTypeHPAV1` instead of `AutoscalingTypeDefault `. if we change the default from HPAV1 to something else then `AutoscalingTypeDefault` will be redundant. Also, we might want to have an option to set the autoscaler explicitly to HPAV1, mainly for customer support workarounds

Reply:
code updated